tests/cupyx_tests/scipy_tests/special_tests/test_zetac.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+import numpy
+
+
+@testing.gpu
+@testing.with_requires('scipy')
+class TestZetac(unittest.TestCase):
+
+    @testing.for_all_dtypes(no_complex=True)
+    @testing.numpy_cupy_allclose(atol=1e-5, scipy_name='scp')
+    def test_arange(self, xp, scp, dtype):
+        import scipy.special  # NOQA
```

Comment:
We'd like to import modules on the top of files whenever it's possible. Would you make it as the following?
https://github.com/cupy/cupy/blob/c8bcae476916985074f5dca4a9f6a5d10458fccd/tests/cupyx_tests/scipy_tests/signal_tests/test_splines.py#L10-L13